neurom/fst/tests/test_feature_compat.py

```diff
@@ -28,6 +28,7 @@
 
 '''Test neurom._point_neurite.features and neurom.fst features compatibility'''
```

Comment:
This is no longer applicable, so the name and docstring should be changed to reflect whatever this test does now.